perses/rjmc/geometry.py

```diff
@@ -525,10 +514,10 @@ def _propose_bond(self, bond, beta):
         """
         Bond length proposal
         """
-        k_eq = bond.type.k*units.kilocalories_per_mole/(units.angstrom**2)
         r0 = bond.type.req*units.angstrom
-        sigma = beta*2.0/np.sqrt(2.0*k_eq/k_eq.unit)
-        r = sigma/sigma.unit*np.random.random()*r0.unit + r0
+        k = bond.type.k*units.kilocalorie_per_mole/units.angstrom**2
```

Comment:
AMBER bond force constants may be off by a factor of two. Check what @swails does here carefully?
https://github.com/ParmEd/ParmEd/blob/master/parmed/structure.py#L2002-L2003